src/test/regress/expected/follower_single_node.out

```diff
@@ -102,15 +113,6 @@ SELECT * FROM ref, local WHERE a = c ORDER BY a;
  7 | 8 | 7 | 8
 (2 rows)
 
--- Check repartion joins are support
-SELECT * FROM test t1, test t2 WHERE t1.x = t2.y ORDER BY t1.x;
-ERROR:  floating-point exception
-DETAIL:  An invalid floating-point operation was signaled. This probably means an out-of-range result or an invalid operation, such as division by zero.
-SET citus.enable_repartition_joins TO ON;
-SELECT * FROM test t1, test t2 WHERE t1.x = t2.y ORDER BY t1.x;
-ERROR:  floating-point exception
-DETAIL:  An invalid floating-point operation was signaled. This probably means an out-of-range result or an invalid operation, such as division by zero.
-RESET citus.enable_repartition_joins;
```

Comment:
Why did you remove these tests (and the ones below)? Is it because they now gave "in recovery" errors? If so, I think it's better to keep them. This way you know they will continue giving the expected error.

Reply:
Yeah I thought the reason we wanted to have those tests were testing repartition joins, not recovery case with repartition joins so that is why I moved them above.

But we can keep them there to test the recovery case for sure, I will put them back before merging this PR.

Reply:
added them back